.github/workflows/publish_and_release.yml

```
@@ -12,59 +12,69 @@ on:
-      uses: elgohr/Github-Release-Action@v5
-      env:
-        GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      with:
-        title: "v${{ steps.get_version.outputs.new_version }}"
-        tag: "v${{ steps.get_version.outputs.new_version }}"
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
```

actionlint (1.7.4)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)